### PR TITLE
package data integration

### LINKED ISSUE #86: climada-petals package does not contain the `data` directory
Contrary to the `climada` package, the `climada-petals` package does not contain the `data` directory of its repository. Is this on purpose or an error?

Compare:

`climada_petals/setup.py`
https://github.com/CLIMADA-project/climada_petals/blob/main/setup.py#L45

`climada/setup.py`
https://github.com/CLIMADA-project/climada_python/blob/main/setup.py#L43

### PR DESCRIPTION
**Pull request by emanuel-schmid**
Changes proposed in this PR:
- Move relevant content of data directory into the package directory

This PR fixes #86

### PR Author Checklist

- [ ] Read the [Contribution Guide][contrib]
- [x] Correct target branch selected (if unsure, select `develop`)
- [ ] Descriptive pull request title added
- [ ] Source branch up-to-date with target branch
- [ ] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#Commenting-&-Documenting) updated
- [ ] [Tests][testing] updated
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]
- [ ] [Changelog](https://github.com/CLIMADA-project/climada_petals/blob/main/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html#Static-Code-Analysis

### CODE REVIEW

**Comment by emanuel-schmid** [comment]
there is a test release on test.pypi.org:

`pip install -i https://test.pypi.org/simple/ climada_petals==3.3.99`

**Comment by emanuel-schmid** [comment]
Most failing tests are because the climada_python develop branch has deviated.

There is one that fails because I didn't move one of the files, mistaking it for obsolete.
I'll fix that.

**Comment by peanutfun** [comment]
> Most failing tests are because the climada_python develop branch has deviated.

These issues are fixed by #88 and #89, right? Then we should merge them first.

**Review by peanutfun — CHANGES_REQUESTED** [review]
I installed the test release and the data is copied correctly. Well done! I just have a few technical comments to the `setup.py`, but since it works, we do not need to dig too deeply into it.

I also ran a couple of tests on the installed version and most seem to work out. However, I found an issue: The `tc_rainfield.py` tries to import `climada.util.tag` (see https://github.com/CLIMADA-project/climada_petals/blob/develop/climada_petals/hazard/tc_rainfield.py#L34), but this has never existed in Climada. How did this escape our tests?

**Comment by emanuel-schmid** [comment]
| These issues are fixed by https://github.com/CLIMADA-project/climada_petals/pull/88 and https://github.com/CLIMADA-project/climada_petals/pull/89, right? Then we should merge them first.

Not if we want to make a patch release that just solves issue #86. This PR targets the main branch, #88 and #89 develop. Since Climada 4 is on the horizon we shouldn't spend too much time here. Just fix the pip release and then make a Conda release, I say.

File: setup.py [hunk 1]
```diff
@@ -42,14 +36,13 @@
...
     install_requires=[
         'climada',
         'scikit-image',
     ],
 
-    package_data={'': extra_files},
+    packages=find_namespace_packages(include=['climada_petals*']),
 
-    include_package_data=True
+    setup_requires=['setuptools_scm'],
```
[the remark this answers is not in the document]
Comment (emanuel-schmid) [reply setup.py:46]: Yeah, I've seent that too. At some point, we'll probably should do that.

File: setup.py [hunk 2]
```diff
@@ -42,14 +36,13 @@
...
 
-    packages=find_packages(where='.'),
-
     install_requires=[
         'climada',
         'scikit-image',
     ],
 
-    package_data={'': extra_files},
+    packages=find_namespace_packages(include=['climada_petals*']),
```
[the remark this answers is not in the document]
Comment (emanuel-schmid) [reply setup.py:44]: Doesn't matter I suppose, The difference is the `__init__.py` files. Adding `__init__.py` files felt awkward to me.

[the remark this answers is not in the document]
Comment (peanutfun) [reply setup.py:44]: I thought we had inits everywhere we needed them. Anyway, it seems to work. Let's keep it as is 👍

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (peanutfun) [reply setup.py:46]: Ok, then leave it like this and keep in mind that we should switch to pyproject.toml at some point

**Comment by peanutfun** [comment]
The changes to the tags in Core are not released yet, right? Then I am fine with merging this now. The tests should then pass when we test the released versions against each other 🤞

**Pull request merged**